### PR TITLE
Adding interface-rename description to Networking Section

### PR DESCRIPTION
**Pull request by jedi-marcus**

### CODE REVIEW

**Comment by stormi** [comment]
Nice addition. Maybe also add that `interface-rename --help` will give you the full list of options, such as `--dry-run` to check your changes before applying, and `--reset-to-install` to cancel your changes. The help also discourages using that command when the interfaces have traffic passing or bonds/bridges/etc.

**Comment by jedi-marcus** [comment]
I'm not the pull request guru... I did some more changes, not sure if you see them here already...

File: docs/networking.md [hunk 1]
```diff
@@ -76,20 +76,34 @@ Plug new NIC:
...
+interface-rename --help
+```
+will display all available options
+
 ```
 interface-rename --list
 ```
 will display the current assignments
 
+The most common use will be an update statment like the following one:
```
Comment (stormi) [inline docs/networking.md:99]: statment => statement

**Review by stormi — APPROVED** [review]

**Comment by olivierlambert** [comment]
Thanks for your contribution @jedi-marcus !

**Comment by olivierlambert** [comment]
Doc is now rebuilt and your change visible at https://xcp-ng.org/docs/networking.html#renaming-nic

**Comment by Fohdeesha** [comment]
Darn, was in the middle of making edits to the source branch :P I'll pull our master down and submit my own PR with small grammar edits

**Comment by olivierlambert** [comment]
Woopsy!

**Pull request merged**